scripts/create-dnsmasq.sh

```diff
@@ -52,10 +52,12 @@ while read -r entry; do
                                 fi
                                 parsed=$(echo ${fileentry} | sed -e "s/^\*\.//")
                                 for i in ${cacheip}; do
-                                        if grep -qx "address=/${parsed}/${i}" "${outputfile}"; then
-                                                continue
+                                        if ! grep -qx "local=/${parsed}/" "${outputfile}"; then
+                                                echo "local=/${parsed}/" >> "${outputfile}"
+                                        fi
+                                        if ! grep -qx "address=/${parsed}/${i}" "${outputfile}"; then
+                                                echo "address=/${parsed}/${i}" >> "${outputfile}"
```

Comment:
```suggestion
                                        if ! grep -qx "address=/${parsed}/${i}" "${outputfile}"; then
                                                echo "address=/${parsed}/${i}" >> "${outputfile}"
                                        fi
                                        if ! grep -qx "local=/${parsed}/" "${outputfile}"; then
                                                echo "local=/${parsed}/" >> "${outputfile}"
```

Can we just swap the order for this, please?